### ISSUE TITLE
IPCM Not detected as pcm_s24be and decoded accordingly by LWLibavAudioSource

### ISSUE THREAD

**Issue by FranceBB** [HolyWu/L-SMASH-Works#20]
Summary of the bug:
IPCM audio files muxed in MP4 are not correctly detected as pcm_s24be and decoded accordingly.

How to reproduce:

```
video=LWLibavVideoSource("Z:\IPCM_Sample.MP4")
audio=LWLibavAudioSource("Z:\IPCM_Sample.MP4")
AudioDub(video, audio)
```

Expected behavior: decode IPCM as pcm_s24be (PCM signed 24-bit big-endian)

Actual behavior: LWLibavAudioSource failed to get audio track

How to workaround:

ffmpeg.exe -c:a pcm_s24be -i "Z:\IPCM_Sample.MP4" -acodec pcm_s24le -ar 48000 -ac 2 -y out1.wav

and then index the newly created PCM signed 24-bit little-endian file muxed in .wav

LSMASH Works version: 20210423
built on 23/04/2021 05:31 AM GMT+1

IPCM Sample: ​https://we.tl/t-UkEK5c94if
Decoded and re-encoded to PCM 24bit little endian audio: ​https://we.tl/t-4yTKInyzmt

**Comment by HolyWu** [comment]
I tried to play `IPCM_Sample.MP4` with MPC-BE and ffplay, and both failed to decode the audio stream.

[![MPC-BE.png](https://i.postimg.cc/m2bHD90d/MPC-BE.png)](https://postimg.cc/Th4wNK2m)

[![ffplay.png](https://i.postimg.cc/VLDbb6wz/ffplay.png)](https://postimg.cc/svG2k31L)

**Comment by FranceBB** [comment]
Yep, I tried with the MPC with LAV as well and it failed, however if you manually specify -c:a pcm_s24be in FFMpeg (and FFPlay for that matters) it works just fine.

**Comment by HolyWu** [comment]
Then I think you should create a ticket on FFmpeg and provide this sample for them to investigate why a proper decoder couldn't be automatically selected.

**Comment by FranceBB** [comment]
Right. Done: https://trac.ffmpeg.org/ticket/9220
I'll keep you posted.
